mentors/python/b.json

```
@@ -38,5 +38,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "Over 6 years Python experience with high Object Orianted skills. Developed Compilers & Restful API's & CLI Apps & Games & Frameworks. Familiar with CPython Source Code (especially Memory Management & Garbage Collecting Techniques) "
+  },
+  {
+    "github_username" : "bubo-py",
+    "name" : "Kamil Chruściński",
+    "link_text" : "My Github",
```

```suggestion
    "link_text" : "My GitHub",
```

Please use correct spelling of GitHub here.